bindings/wasm/package.json

```diff
@@ -11,8 +11,8 @@
   },
   "scripts": {
     "build:src": "cargo build --lib --release --target wasm32-unknown-unknown",
-    "bundle:nodejs": "wasm-bindgen target/wasm32-unknown-unknown/release/identity_wasm.wasm --typescript --weak-refs --target nodejs --out-dir node && node ./build/node && tsc --project ./tsconfig.node.json --outDir node",
-    "bundle:web": "wasm-bindgen target/wasm32-unknown-unknown/release/identity_wasm.wasm --typescript --weak-refs --target web --out-dir web && node ./build/web && tsc --project ./tsconfig.web.json --outDir web",
+    "bundle:nodejs": "wasm-bindgen target/wasm32-unknown-unknown/release/identity_wasm.wasm --typescript --weak-refs --target nodejs --out-dir node && node ./build/node && tsc --project ./lib/tsconfig.json && tsc-alias -p ./lib/tsconfig.json",
+    "bundle:web": "wasm-bindgen target/wasm32-unknown-unknown/release/identity_wasm.wasm --typescript --weak-refs --target web --out-dir web && node ./build/web && tsc --project ./lib/tsconfig.web.json && tsc-alias -p ./lib/tsconfig.web.json",
```

Comment:
This PR moved and renamed the `tsconfig.node.json` file, the other scripts using `ts-mocha` in `package.json` need to be updated with the new path: `test:account:node`, `test:stardust`, `test:unit:node`.

Reply:
This actually breaks all the tests even after changing the path to `tsconfig`, I don't know how to run `tsc-alias` with `ts-mocha`.

```
  1) Uncaught error outside test suite:
     Uncaught Cannot find module 'identity_wasm'
```